examples/toy_npe_model/asd_dataset_settings.yaml

```diff
@@ -3,7 +3,7 @@ dataset_settings:
 #  f_max: 2048      # defaults to f_s/2
   f_s: 4096
   time_psd: 1024
-  T: 8
+  T: 4
```

Comment:
We're using 4 s PSDs here; is that consistent with the rest of the "toy example" settings?

Reply:
Right, it was changed to 8 in the last commit which broke the example. So I changed it back to 4 to get things to work.